src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -360,13 +360,14 @@ export default class ArrivalBase {
                     // if point before next fix
                     const nextFix = waypoint;
                     const previousFix = waypointModelList[j - 1];
-                    const heading = bearing(previousFix.gps, nextFix.gps);
-                    const calculatedRadialDistance = fixRadialDist(previousFix.gps, heading, spawnOffset);
+                    const heading = bearingToPoint(previousFix.gps, nextFix.gps);
+                    const spawnPoint = fixRadialDist(previousFix.gps, heading, spawnOffset).reverse();
```

Comment:
Received `[longitude, latitude]`, because `PositionModel.gps` returns in [x,y] order, and expected `[latitude, longitude]`. Addressed this in recent revisions to the PR.